<?xml version="1.0"?>
<launch>

    <node pkg="drone_navigation" name="drone_marker" type="drone_marker.py" output="screen" />

    <node pkg="octomap_server" type="octomap_server_node" name="octomap_server" args="$(find drone_navigation)/map/lab.ot">
          <param name="resolution" value="0.5" />
</node>
<!--
    <node pkg="drone_navigation" name="path_planner" type="path_planner" />
-->

<!--
    <node pkg="path_planner_3d" name="path_planner_3d" type="path_planner_3d_node">
    	<param name="max_iter" value="3000" />
    	<param name="step_size" value="0.1" />
    	<param name="end_dist_threshold" value="0.01" />
    	<param name="ground_robot" value="false" />
    </node>
-->

    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find drone_navigation)/rviz/octomap.rviz" output="screen" />

</launch>
